test/forge/HalmosTest.sol

```diff
@@ -182,7 +173,7 @@ contract HalmosTest is SymTest, Test {
         _callMorpho(selector, caller);
 
         uint256 nonceAfter = morpho.nonce(user);
-        assert(nonceAfter == nonceBefore || nonceAfter == nonceBefore + 1);
+        assert(nonceAfter == nonceBefore);
```

Comment:
why did you change this? it cannot pass like that right?

Reply:
fixed in da3ebee

Reply:
Oops my bad, I wanted to look at the error message and I committed this by accident